chart/flux/templates/deployment.yaml

```diff
@@ -160,7 +160,7 @@ spec:
           - --registry-ecr-exclude-id={{ .Values.registry.ecr.excludeId }}
           {{- end }}
           {{- if .Values.registry.dockercfg.enabled }}
-          - --docker-config=/dockercfg/config.json
+          - --docker-config="{{ .Values.registry.dockercfg.configFileName }}"
```

Comment:
I don't think we need quotes here.

Reply:
I did this way so to be consistent with how value is passed for this other parameter:
```
hostPath:
  path: "{{ .Values.registry.acr.hostPath }}"
```

Reply:
Good point.. will update our change here but not going to fix other parts of the template

Reply:
The reason being, the version you've done will surround with quotes even if the value originally contains quotes, resulting in `""<hostPathValue""` By using the `| quote` syntax, it will only place surrounding quotes if not already present